src/test/java/br/unicamp/cst/io/rest/TestREST.java

```diff
@@ -0,0 +1,266 @@
+/***********************************************************************************************
+ * Copyright (c) 2012  DCA-FEEC-UNICAMP
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the GNU Lesser Public License v3
+ * which accompanies this distribution, and is available at
+ * http://www.gnu.org/licenses/lgpl.html
+ * <p>
+ * Contributors:
+ * K. Raizer, A. L. O. Paraense, E. M. Froes, R. R. Gudwin - initial API and implementation
+ * **********************************************************************************************/
+package br.unicamp.cst.io.rest;
+
+import br.unicamp.cst.core.entities.Codelet;
+import br.unicamp.cst.core.entities.Memory;
+import br.unicamp.cst.core.entities.MemoryContainer;
+import br.unicamp.cst.core.entities.MemoryObject;
+import br.unicamp.cst.core.entities.Mind;
+import br.unicamp.cst.support.InterfaceAdapter;
+import br.unicamp.cst.support.TimeStamp;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+/**
+ *
+ * @author rgudwin
+ */
+public class TestREST {
+    
```

Comment:
This test is not really asserting anything, it is just running the code. Would be nice if we had proper test cases, with assertions.

Reply:
@andre-paraense, even though the test is not exhaustive, if you see in lines 225-228, we test the use of the code and assert the obtained data is the correct data flowing within the mind.